pf/tests/internal/cmd/pulumi-resource-testbridge/schema.json

```diff
@@ -231,6 +256,13 @@
                 "statedir": {
                     "type": "string",
                     "description": "Dir to store pseudo-cloud state in.\n"
+                },
+                "tuplesOptionals": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/types/testbridge:index/TestresTuplesOptional:TestresTuplesOptional"
```

Comment:
Ah every tuple will generate a named type! That's not ideal.. I think we also have type merging in tfgen. Two structurally identical types will possibly collapse to the same type name so maybe that's good. The name itself is weird though because ideally it'd be a system type. Regardless given our conversations on lack of evidence of prevalence we can shelve this till we find actual use cases.